app/assets/javascripts/host_edit.js

```diff
@@ -410,19 +410,24 @@ function override_param(item){
 
 function override_class_param(item){
   var param = $(item).closest('tr[id^="puppetclass_"][id*="_params\\["][id$="\\]"]');
-  var id = param.attr('id').replace(/puppetclass_\d+_params\[(\d+)\]/, '$1')
+  var id = param.attr('id').replace(/puppetclass_\d+_params\[(\d+)\]/, '$1');
   var c = param.find('[data-property=class]').text();
```

Comment:
Kind of unrelated and nitpicking, but would you mind to change these 1-letter variables to something more meaningful?